homeassistant/components/suez_water/.coveragerc

```diff
@@ -0,0 +1 @@
+homeassistant/components/suez_water/*
```

Comment:
Remove this file as it's already covered in ` .coveragerc`.

Reply:
This hasn't been resolved. The file is still here.

Reply:
You're right. Forgot to remove it. Done.